guide/english/javascript/index.md

```diff
@@ -5,7 +5,7 @@ title: JavaScript
 ## JavaScript
 JavaScript is the most widely used scripting language on Earth. And it has the largest library ecosystem of any programming language.
 
-JavaScript is the core language of the web, and the only programming language that can run in all major web browsers.
+JavaScript is the core language of the web, and the only programming language that can run in all major web browsers. It is used to add more creative and entertaining content to the site by introducting animation and visul effects. It helps your website look more lively.
```

Comment:
I do not think this ` It is used to add more creative and entertaining content to the site by introducting animation and visul effects. It helps your website look more lively.` adds any value to existing content. Please remove it.

Reply:
Thanks for the feedback. I will change it accordingly